tgg/research/gnomad_tgps.py

```diff
@@ -194,28 +239,33 @@ def compute_stats(
     )
     mt_filtered = mt.anti_join_rows(pg_ht)
 
-    # Annotate the SNV filtered gnomAD dataset to count the number of minor alleles
+    # Annotate the SNV filtered gnomAD dataset to count the number of common variants
     # carried by each participant.
     mt_filtered = mt_filtered.annotate_cols(
         n_var_per_sample=hl.agg.count_where(mt_filtered.GT.is_non_ref())
     )
 
-    # Keep only the columns and annotate with population data, both continental and
-    # subcontinental ancestry.
+    # Keep only the columns and annotate with genetic ancestry group labels.
     ht = mt_filtered.cols()
-    ht = ht.annotate(population=meta_ht[ht.key].population_inference.pop)
+    ht = ht.annotate(genetic_ancestry=meta_ht[ht.key].population_inference.pop)
+    # ##? naive_coalesce does not seem to reduce the number of partitions here.
+    # The checkpoint still uses over 115,000 partitions.
```

Comment:
does the HT that gets written as part of this checkpoint have fewer partitions? it's normal to see huge numbers while this is checkpointing, since Hail is starting from the total number of partitions present in the input MT, then running the desired filters/annotations prior to checkpointing.

if the HT that gets written out doesn't have fewer partitions, then I would do a little more digging/save the Hail log and reach out to the Hail team if the problem is reproducible